source/_integrations/homewizard.markdown

```diff
@@ -79,11 +79,11 @@ You can also control the green status light brightness with **Status light brigh
 ## Identify
 
 The identify button can be pressed to let the status light blink for a few seconds.
-This feature is currently only available for the Wifi Energy Socket with firmware version `3.01` or later.
+This feature is currently only available for the Wifi P1 meter with firmware version `4.00` or later and the Wifi Energy Socket.
```

Comment:
_Almost_ all energy sockets are updated so there is no need anymore to tell about the firmware requirement.